src/Forcings/continuous_forcing.jl

```diff
@@ -83,21 +83,21 @@ callable with the signature
 ContinuousForcing(func; parameters=nothing, field_dependencies=()) =
     ContinuousForcing(func, parameters, field_dependencies)
 
-""" 
-    regularize_forcing(forcing::ContinuousForcing, field_name, model_field_names)
+"""
+    regularize_forcing(forcing::ContinuousForcing, field, field_name, model_field_names)
 
 Regularize `forcing::ContinuousForcing` by determining the indices of `forcing.field_dependencies`
 in `model_field_names`, and associated interpolation functions so `forcing` can be used during
 time-stepping `IncompressibleModel`.
 """
-function regularize_forcing(forcing::ContinuousForcing, field_name, model_field_names)
+function regularize_forcing(forcing::ContinuousForcing, field, field_name, model_field_names)
```

Comment:
Doesn't look like `field_name` is used anymore? It should be removed from the function signature if so.

Reply:
Good catch, thanks. I thought it was used by

```julia
regularize_forcing(forcing::Relaxation, field, field_name, model_field_names)
```

but upon closer inspection I don't think it's used anywhere anymore.

Reply:
Ah wait never mind it's used in

https://github.com/CliMA/Oceananigans.jl/blob/032ceee13b46b561d524c3ece51f6f5a98d3176d/src/Forcings/relaxation.jl#L77

so I reverted my commit.